src/main/java/step4/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package step4.domain;
+
+public class Car {
+    private final int condition = 4 ;
+    Name name;
+    Position position;
+
+    public Car(String name) {
+        this(new Name(name), new Position());
+    }
+
+    public Car(Name name, Position position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public void move(int randomValue) {
+        if(condition < randomValue){
+            position.plus();
+        }
+    }
+
+    public boolean isMoving() {
+        return myPosition() > 0;
+    }
+
+    public int myPosition() {
+        return position.getPosition();
+    }
+
+    public String myName() {
+        return name.getName();
+    }
```

Comment:
```suggestion
    public int position() {
        return position.getPosition();
    }

    public String name() {
        return name.getName();
    }
```

my 라는 수식어는 딱히 의미를 가지지 못하므로, 없애도 무방하지 않나 제안해봅니다.